lib/routific.rb

```diff
@@ -53,10 +66,19 @@ def token
       @@token
     end
 
+    def endpoint
+      @@endpoint
+    end
+
+    def endpoint=(value)
+      raise InvalidEndpoint unless ENDPOINTS.include?(value.to_sym)
```

Comment:
this is repeated between the class side method and the instance side method, might be worth extracting :)